src/lib/tabs/Tabs.tsx

```diff
@@ -70,9 +71,9 @@ export const Tabs: React.FC<{ ariaLabel?: string; tabs: { label: string; render:
   };
 
   return (
-      <div className={classes.root}>
-      <AppBar className={classes.appBar} position="static">
-        <MuiTabs classes={{indicator: classes.indicator}} value={value} onChange={handleChange} aria-label={ariaLabel}>
+    <div className={classes.root}>
+      <AppBar className={classes.appBar} color='default' position='static'>
```

Comment:
is color default needed? seems like that would be assumed

Reply:
Lol yup! The default of `color` is actually `primary`. Checkout the API: https://material-ui.com/api/app-bar/ 

Without it set to default, the color of the arrows is white and they're not visible to user when they're active.

Reply:
haha, that's kinda funny, default is not default